diff --git a/node_modules/@notionhq/client/build/src/Client.d.ts b/node_modules/@notionhq/client/build/src/Client.d.ts
index 540eafa..e3085af 100644
--- a/node_modules/@notionhq/client/build/src/Client.d.ts
+++ b/node_modules/@notionhq/client/build/src/Client.d.ts
@@ -11,7 +11,7 @@ export interface ClientOptions {
     logLevel?: LogLevel;
     logger?: Logger;
     notionVersion?: string;
-    fetch?: SupportedFetch;
+    fetch: SupportedFetch;
     /** Silently ignored in the browser */
     agent?: Agent;
 }
diff --git a/node_modules/@notionhq/client/build/src/Client.js b/node_modules/@notionhq/client/build/src/Client.js
index 390de9c..8faf72a 100644
--- a/node_modules/@notionhq/client/build/src/Client.js
+++ b/node_modules/@notionhq/client/build/src/Client.js
@@ -16,7 +16,6 @@ const logging_1 = require("./logging");
 const errors_1 = require("./errors");
 const utils_1 = require("./utils");
 const api_endpoints_1 = require("./api-endpoints");
-const node_fetch_1 = require("node-fetch");
 const package_json_1 = require("../package.json");
 class Client {
     constructor(options) {
@@ -312,7 +311,7 @@ class Client {
         __classPrivateFieldSet(this, _Client_prefixUrl, ((_c = options === null || options === void 0 ? void 0 : options.baseUrl) !== null && _c !== void 0 ? _c : "https://api.notion.com") + "/v1/", "f");
         __classPrivateFieldSet(this, _Client_timeoutMs, (_d = options === null || options === void 0 ? void 0 : options.timeoutMs) !== null && _d !== void 0 ? _d : 60000, "f");
         __classPrivateFieldSet(this, _Client_notionVersion, (_e = options === null || options === void 0 ? void 0 : options.notionVersion) !== null && _e !== void 0 ? _e : Client.defaultNotionVersion, "f");
-        __classPrivateFieldSet(this, _Client_fetch, (_f = options === null || options === void 0 ? void 0 : options.fetch) !== null && _f !== void 0 ? _f : node_fetch_1.default, "f");
+        __classPrivateFieldSet(this, _Client_fetch, options === null || options === void 0 ? void 0 : options.fetch, "f");
         __classPrivateFieldSet(this, _Client_agent, options === null || options === void 0 ? void 0 : options.agent, "f");
         __classPrivateFieldSet(this, _Client_userAgent, `notionhq-client/${package_json_1.version}`, "f");
     }
diff --git a/node_modules/@notionhq/client/build/src/utils.js b/node_modules/@notionhq/client/build/src/utils.js
index 3a71c2d..40e4b1e 100644
--- a/node_modules/@notionhq/client/build/src/utils.js
+++ b/node_modules/@notionhq/client/build/src/utils.js
@@ -14,7 +14,10 @@ function assertNever(value) {
 exports.assertNever = assertNever;
 function pick(base, keys) {
     const entries = keys.map(key => [key, base === null || base === void 0 ? void 0 : base[key]]);
-    return Object.fromEntries(entries);
+    return entries.reduce((obj, [key, value]) => {
+      obj[key] = value;
+      return obj;
+    }, {});
 }
 exports.pick = pick;
 function isObject(o) {
