diff --git a/node_modules/@notionhq/client/build/src/errors.js b/node_modules/@notionhq/client/build/src/errors.js
index e8092b2..0bf7d1f 100644
--- a/node_modules/@notionhq/client/build/src/errors.js
+++ b/node_modules/@notionhq/client/build/src/errors.js
@@ -65,13 +65,13 @@ class RequestTimeoutError extends NotionClientErrorBase {
     }
     static rejectAfterTimeout(promise, timeoutMS) {
         return new Promise((resolve, reject) => {
-            const timeoutId = setTimeout(() => {
+            const timeoutId = Zotero.setTimeout(() => {
                 reject(new RequestTimeoutError());
             }, timeoutMS);
             promise
                 .then(resolve)
                 .catch(reject)
-                .then(() => clearTimeout(timeoutId));
+                .then(() => Zotero.clearTimeout(timeoutId));
         });
     }
 }
