diff --git a/node_modules/node-fetch/browser.js b/node_modules/node-fetch/browser.js
index 83c54c5..e60b420 100644
--- a/node_modules/node-fetch/browser.js
+++ b/node_modules/node-fetch/browser.js
@@ -5,6 +5,7 @@ var getGlobal = function () {
 	// the only reliable means to get the global object is
 	// `Function('return this')()`
 	// However, this causes CSP violations in Chrome apps.
+	if (typeof Zotero !== 'undefined') { return Zotero.getMainWindow(); }
 	if (typeof self !== 'undefined') { return self; }
 	if (typeof window !== 'undefined') { return window; }
 	if (typeof global !== 'undefined') { return global; }
